besu/src/main/java/org/hyperledger/besu/controller/BesuControllerBuilder.java

```diff
@@ -100,6 +100,7 @@
 import java.util.OptionalLong;
 import java.util.function.Supplier;
 
+import org.jetbrains.annotations.NotNull;
```

Comment:
Did you mean javax.validation.constraints.NotNull ?  the jetbrains annotation only has an effect in the IDE afaik.

Reply:
removed per convo with @jflo